host_source.go

```diff
@@ -415,13 +415,6 @@ func (h *HostInfo) IsUp() bool {
 	return h != nil && h.State() == NodeUp
 }
 
-func (h *HostInfo) HostnameAndPort() string {
```

Comment:
This function became unused so I removed it for cleanliness

Reply:
Unfortunately, removing the function breaks backwards compatibility - both `HostInfo` and `HostnameAndPort` are exported, so `HostnameAndPort` could be used by some gocql user. So we need to keep it.